src/sentry/api/serializers/models/integration.py

```diff
@@ -189,6 +189,7 @@ def get_attrs(self, item_list, user, *args, **kwargs):
             issues_by_integration[ei.integration_id].append({
                 'id': six.text_type(ei.id),
                 'key': ei.key,
+                'url': [i.get_installation(self.group.organization.id).get_issue_url(ei.key) for i in item_list if i.id == ei.integration_id][0],
```

Comment:
to make this more efficient, you could do something outside of the loop like: `ints_by_id = {i.id: i for i in item_list}` and then inside the loop, you could do `'url': ints_by_id[ei.integration_id].get_installation(self.group.organization.id).get_issue_url(ei.key),` do you don't have to iterate over the whole item_list every time.

(this can just be a follow up)